src/main/resources/messages_en_GB.properties

```diff
@@ -674,6 +674,7 @@ pageLayout.submit=Submit
 scalePages.title=Adjust page-scale
 scalePages.header=Adjust page-scale
 scalePages.pageSize=Size of a page of the document.
+scalePages.keepPageSize=Keep source file Page Size
```

Comment:
I think adding ``source file`` etc is not needed, and makes it to long compared to the A1 sizes etc and more complex sounding
Lets just keep it as ``Original Size`` or ``Unchanged``